img/drop.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid" width="82" height="82" viewBox="0 0 82 82">
```

Comment:
Although I think it makes sense to have this in the project because ideally you'd be able to load it in to your HTML document with `svg use xlink`, try not to have unused assets left in your project. It ultimately ends up leading to confusion.